indsl/signals/user_specified_timeseries.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Cognite AS
+from typing import List
+
+import pandas as pd
+
+from indsl.exceptions import UserValueError
+
+
+# from indsl.validations import validate_series_is_not_empty
+
+
+# @check_types
```

Comment:
Check types is required for functions that should be exposed in the UI. Thin function checks that the provided types are matching expected types.